### DISCUSSION TITLE
Rule for adding quote (backtick) to table and column names for MySQL queries?

### DISCUSSION THREAD

**Discussion by Nlhmmh** [aarondl/sqlboiler#1368] [Q&A] [answered]
I am wondering whether sqlboiler always adds `quote (backtick) to table and column names `for generated MySQL queries.
```
SELECT * FROM `my_db`.`my_table`;
```
Is it like a general rule or something decided behind the scene?
I could not find anything related in `README`.

I would appreciate if someone can answer. 🙏

**Comment by stephenafamo** [comment, marked as answer]
SQLBoiler tries to add the quotes all the time, because without it, mixed-case names or names that clash with keywords will cause errors

**Reply by Nlhmmh** [reply]
Thank you for your answer.
